app/config/services/platform.xml

```diff
@@ -100,6 +100,10 @@
             <tag name="validator.constraint_validator" alias="app.validator.unique_membership"/>
         </service>
 
+        <service id="AppBundle\Form\ColorChoiceType">
```

Comment:
You could try to add these to the new `services.xml` where autoconfigure and autowired are enabled.